### ISSUE TITLE
Я-стрелка с белым фоном

### ISSUE THREAD

**Issue by arteg** [yandex-ui/stylobate-islands#61]
Как здесь в шапке https://passport.yandex.ru/auth

Еще у них там вроде бы треугольник справа хорошо кроссбраузерно сделан, и его высота зависит от высоты стрелки)

**Comment by kizu** [comment]
Да, нужно сделать стрелку с бордером, в общем. Тот вариант, что там — не совсем честный, он работает только над белым/ровным фоном, так как работает за счёт перекрытия прямоугольного блока элементом, делающим видимость стрелки.
